.mergify.yml

```diff
@@ -125,9 +125,12 @@ pull_request_rules:
           * `backport-v./d./d./d` is the label to automatically backport to the `8./d` branch. `/d` is the digit
 
           **NOTE**: `backport-skip` has been added to this pull request.
+
+          `backport-v8.x` has been added to help with the transition to the new branch 8.x.
       label:
         add:
           - backport-skip
```

Comment:
Should this line be removed?  Otherwise won't we end up with `backport-skip` and `backport-8.x` both?

Reply:
Good question. There is a Mergify task to remove `backport-skip` when `backport-v` is added (see L134). I initially considered removing it, but I've realised it might be needed once `8.x` becomes like `7.17`.

As you prefer, I have no strong opinion; if we need that in the future, it can be re-added.